_pages/platform/algorithm-secrets.md

```diff
@@ -29,34 +30,45 @@ This feature is available to [Algorithmia Enterprise](/enterprise) users only.
 
 Storing sensitive data on Algorithmia might not be suitable for an organization that uses a vaulting system like Hashicorp Vault or Azure Key Vault to manage secrets.
 
-Algorithmia Enterprise allows platform admins to connect Algorithmia to external vaulting systems where secrets are managed and maintained by a DevOps team. This helps Data Science teams access secrets in accordance to an organization's security and compliance standards.
-
-If you're new to Algorithmia and would like to learn more about our product and using external secret stores, please [contact our sales team](https://info.algorithmia.com/contact-sales). We'd love to hear from you!
+Algorithmia Enterprise's secret store enables cluster admins to connect Algorithmia to external vaulting systems where secrets are managed and maintained by a DevOps team. This helps Data Science teams access secrets in accordance to an organization's security and compliance standards.
```

Comment:
@williamhansen any changes you want to make here?

Reply:
I want to be sure that it's clear that customers can integrate their Vault tool with Algorithmia's Secret Store.

Reply:
changed wording to clarify.